### PR TITLE
Adds finishing touches to my portfolio page by designing the Buttons, and Form fields.

### PR DESCRIPTION
**Pull request by Cayle**
Implemented CSS and client-side JavaScript to my web page so it looks more attractive.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -86,7 +86,23 @@ function changePattern(pattern) {
     }
   }
 }
+function createCommentElement(eachComment) {
+    const divElement = document.createElement('div');
```
Comment (rayliu0317) [inline portfolio/src/main/webapp/script.js:90]: Should be consistent with the usage of single or double quotes, because you have used double quotes below.

Comment (rayliu0317) [inline portfolio/src/main/webapp/script.js:90]: I don't really see why you would need to make this a const here. var is used in a similar way in the below function. It's definitely fine, but would like to hear your thought on this.

Comment (Cayle) [reply portfolio/src/main/webapp/script.js:90]: Most times, I use const for things I don't plan to change or update while the program is running. Basically, that's how I look at it. I don't know if that is right or not.

Comment (rayliu0317) [reply portfolio/src/main/webapp/script.js:90]: You are mostly right. But I think you can think of this like this: divElement points to a variable, making it const will prevent the program from making it points to another variable. But you can still modify and update it by accessing its fields just like you did below.

You are still using it correctly though...just FYI:)

Comment (Cayle) [reply portfolio/src/main/webapp/script.js:90]: Oh, okay. I'll take note of that.

Comment (Cayle) [reply portfolio/src/main/webapp/script.js]: I have just modified the strings to use double quotes.

**Pull request merged**